operator/src/network/ipfs.rs

```diff
@@ -110,6 +117,7 @@ impl Default for RustIpfsConfig {
                 memory: Some(Quantity("512Mi".to_owned())),
                 storage: Quantity("1Gi".to_owned()),
             },
+            storage_class: None,
```

Comment:
I'm not sure exactly how this will evaluate with Rust.

If there is no storageClassName specified, then the PVC should not set one, not even an empty string, so that the default storageClassName is applied.

https://kubernetes.io/docs/concepts/storage/persistent-volumes/#class-1

Reply:
That is what is happening, if no storage class is specified in the spec then its not set on the PVC spec itself. Rust does this via the `Option<String>` type, where `None` means its not set and `Some("some_storage_class_name")` is set.